front/src/component/cocktailSearch/CocktailDetailSearch.js

```diff
@@ -17,13 +18,17 @@ const CocktailDetailSearch = (props) => {
   });
 
   const onLoadCocktail = async () => {
-    const response = await fetchCocktail(id);
-    const { data } = response;
-    setCocktailData({
-      cocktail: data,
-      tags: data.tags,
-      recipe: data.recipe,
-    });
+    try {
+      const response = await fetchCocktail(id);
```

Comment:
catch할 에러가 fetch 에서 발생한 것이라면
```suggestion
      const response = await fetchCocktail(id)
                        .then(response => 성공시 로직)
                        .catch(err => 에러 처리);
```
이런 방식으로 하면 조금 더 깔끔하지 않을까요?
혹시 try catch로 잡는 이유가 따로 있는건가요? 아니면 그니의 스타일인가요?
궁금합니다! 😄